tools/pd-simulator/simulator/task.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2017 TiKV Project Authors.
+// Copyright 2022 TiKV Project Authors.
```

Comment:
why change it?

Reply:
Caused by the newly created file, it has been rolled back.